scripts/src/osmo-test.ts

```diff
@@ -291,22 +292,27 @@ async function claim(
   const sigObj = secp256k1.ecdsaSign(msg, private_key);
   const signature = Buffer.from(sigObj.signature).toString("base64");
 
-  // Generate cosmwasm 'binary' base64 string from amount array
-  let amounts_string = amounts.map((amt) => amt.toString()).join(",");
-  const claim_amounts_string = Buffer.from(amounts_string).toString("base64");
+  let amounts_string_array = amounts.map((amt) =>
+    amt.toLocaleString("fullwide", { useGrouping: false })
```

Comment:
Note that `.toLocaleString("fullwide", { useGrouping: false })` and `.toString()` are similar